pages/docs/concepts/asyncapi-document/dynamic-channel-names.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Dynamic Channel Names
```

Comment:
We talk about dynamic addresses. In previous versions, the address was also the name of the channel, but in v3, as I mentioned in https://github.com/asyncapi/website/pull/1959#discussion_r1265651267, it's moved into the `address` field.

This should be changed across the whole doc. 
```suggestion
title: Dynamic Channel Addresses
```